### DISCUSSION TITLE
Unknown issues and PRs in inbox

### DISCUSSION THREAD

**Discussion by yaroslavyaroslav** [community/community#121375] [Other Feature Feedback, Questions, & Ideas] [answered]
Today I opened GitHub client and saw a list of completely unknown to me  items. I mean literally each and every item from this list is unknown to me. More than that I believe that at least one item could be from a private repo (it shows 404 on a click on it). 

![image](https://github.com/community/community/assets/16612247/7d27fdd9-a785-4189-9d8c-576e8fb45a16)
![image](https://github.com/community/community/assets/16612247/8079aed9-0ee5-450d-b2e8-bb5bcaa3e378)

**Comment by gil-son** [comment, marked as answer]
Hello, how are you? I feel for that situation, you need to verify it


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!
